cantools/database/can/signal.py

```diff
@@ -305,6 +321,41 @@ def __init__(self,
             # multi-lingual dictionary
             self.comments = comment
 
+    @property
+    def initial(self) -> Optional[Union[int, SignalValueType]]:
+        warnings.warn(
```

Comment:
will this produce a warning every time the property is accessed or only for the first time? (IOW, how annoying do we want to make these warnings?)

also: `.invalid`, `.minimum`, and `.maximum` need the same treatment.

Reply:
This is obsolete since invalid, minimum and maximum will not be renamed.